exercises/practice/gigasecond/.docs/instructions.md

```diff
@@ -1,5 +1,8 @@
 # Instructions
 
-Given a moment, determine the moment that would be after a gigasecond has passed.
+Your task is to determine the date and time one gigasecond after a certain date.
```

Comment:
Does this too strongly suggest that we should get a Time or Date object back?  It is not required, but used for a few reasons in the tests themselves.

If the unit were not so large, a Degree would also work since the base unit of a degree is a second, and integer solutions work well (as do float solutions) because the base unit of Time is the second, and it is represented as a `Numeric`.

Reply:
No, I don't think this says anything about how the date and time is implemented.

Reply:
That is true, but now uses date and time language specifically, where before it suggested only the _simple_ idea of a gigasecond more than a number of seconds given.

So, does it suggest that we should use date or time specifically?  Humans tend to easily read too much into statements as it is.  Rather than discarding irrelevant information, which often helps in solving problems.

But it does not really matter, since these things can be addressed via mentoring.  I already talk about the potential "reading too much into" which often happens due to the tests choice to use Time here, rather than the simpler addition of an integer solution that would and does still work.

Reply:
I chose to say "date and time" very explicitly, because that is how we talk about these things in the real world. This is an attempt to ensure that the language used is more in the realm of day-to-day conversation than in the realm of programming syntax.

I'm completely okay with people reading too much into it. People do that all the time no matter what we do and say. It will happen in their day-to-day life as well. We do not need to take every possible opportunity to ensure to the best of our ability that this does not happen in this simple exercise.

And, as you say, if people choose to be mentored, this is something that can be raised then.

Reply:
I can work with it either way.   This adds more fodder to the aspect of the mentoring in my snippets about "Boss Speak" which sometimes hold misdirection to solutions.

Sounds like more fun in mentoring this way. ;)